### ISSUE TITLE
Don't trigger autocorrection on backspace.

### ISSUE THREAD

**Issue by matter123** [genesy/auto-correct#15]
When auto-correct corrects words when backspacing it becomes impossible to delete the word via the backspace key.

Steps to reproduce:

1. use the large set
2. open a new document
3. type the word `there`
4. press backspace in an attempt to remove the last `e`
5. Note that `ther` has ben auto corrected to `there`, undoing the action of step 4

**Comment by aleclarson** [comment]
Yeah this one's really annoying. One way to detect the backspace would be to compare previous document length with current.
